sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/action/ReflectiveActionRouter.scala

```diff
@@ -43,11 +47,15 @@ class ReflectiveActionRouter[A <: Action](action: A, commandHandlers: Map[String
         message.metadata())
 
     val inputTypeUrl = message.payload().asInstanceOf[ScalaPbAny].typeUrl
-
-    commandHandler
-      .lookupInvoker(inputTypeUrl)
-      .invoke(action, context)
-      .asInstanceOf[Action.Effect[_]]
+    val methodInvoker = commandHandler.lookupInvoker(inputTypeUrl)
+
+    methodInvoker match {
+      case Some(invoker) =>
+        invoker
+          .invoke(action, context)
+          .asInstanceOf[Action.Effect[_]]
+      case None => ActionEffectImpl.Builder.ignore()
```

Comment:
Usually topic/entity subscriptions are done in with unary calls. I'm not sure if it will work on streamed calls. Possibly. In any case, we need to protect this method with the `ignoreUnknown` flag.

The tests are passing because we are always ignoring. We should only ignore if the flag is true. We should have a test that fails.

Reply:
wow. Totally missed that `if`. 

That's a good point. I tried to create that test but it turns out that we can't catch that exception because gets thrown in another thread. I can do a test but it is very ugly and will need a brand-new entity. Shall I?

Reply:
regarding the testing created the following [issue](https://github.com/lightbend/kalix/issues/7756)